lua/nvim-lsp-installer/servers/perlnavigator/init.lua

```diff
@@ -0,0 +1,20 @@
+local server = require "nvim-lsp-installer.server"
+local path = require "nvim-lsp-installer.path"
+local npm = require "nvim-lsp-installer.installers.npm"
+
+return function(name, root_dir)
+    return server.Server:new {
+        name = name,
+        root_dir = root_dir,
+        languages = { "perl" },
+        homepage = "https://github.com/bscan/PerlNavigator",
+        installer = npm.packages { "perlnavigator-server" },
+        default_options = {
+            cmd = {
```

Comment:
I think it'd be neat if the npm package shipped with an executable. Not sure if you have any connection with the maintainer(s) of this server, but could be worthwhile bringing up with them

Reply:
It would be neat, but for now this is what is available. I modeled it after the lspconfig for rescriptls since it seems to be similar it that it has no executable.

Reply:
@bscan any opinion on whether this can/will be be done? Any interest on your end?

Reply:
Yes, this is planned.  I committed some changes that utilize https://github.com/vercel/pkg to build a single executable file with no external dependencies.  If you run "npm install" and then "npm run build-binary", then it builds binaries for Linux, OS X, and Windows.  I'm going to talk to @bscan about adding some git releases with the downloadable executables.  Would that work better for you?  I'm trying to get this to work easily for Emacs, but I think the needs are similar.  It seems silly to require a full nodejs installation for a user trying to run a Perl language server.

Reply:
@ketbra I think what @williamboman was talking about isn't so much a binary (although maybe he was) as a single name that does the right thing. Examples of nodejs servers that do this: bash-language-server, ansible-language-server, and vim-language-server. There are many others. Each generates a name that is typically a symlink to a js file that knows how to load the server.  e.g.  bash-language-server's build process creates a 'bash-language-server' that symlinks to, I think, main.js. Building an _actual_ binary, as you are proposing, is interesting. I imagine the installer would just have to download it, probably using the `std` installer provided by nvim-lsp-installers.

Reply:
@ketbra just tried it with most recent `main`. That's pretty nice. If you can publish them automatically to github that's even better. Once that happens I can change how the installer works.  I'll still need `node` unfortunately since I have  many other LSP servers that use it. :(

Reply:
Exactly, really just a script that runs node for you. Usually these are just `#!/usr/bin/env node` scripts that provide a convenient CLI for the package - see https://github.com/bash-lsp/bash-language-server/blob/master/server/bin/main.js for example. Shipping standalone binaries is an interesting prospect as well!

Reply:
Gotcha.  That makes sense.  If you do decide to switch to the standard installer, you can now grab the latest releases at the links https://github.com/bscan/PerlNavigator/releases/latest/download/perlnavigator-(?:linux.tar.gz|macos.tar.gz|win.zip)  The benefit of that being that a user doesn't need to install Node in order to use a Perl language server.